lib/clevertap_plugin.dart

```diff
@@ -755,71 +773,110 @@ class CleverTapPlugin {
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Fetches the Product Configs from CleverTap
   static Future<void> fetch() async {
     return await _dartToNativeMethodChannel.invokeMethod('fetch', {});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Fetches Product configs, adhering to the specified minimum fetch interval in seconds.
   static Future<void> fetchWithMinimumIntervalInSeconds(int interval) async {
     return await _dartToNativeMethodChannel.invokeMethod(
         'fetchWithMinimumFetchIntervalInSeconds', {'interval': interval});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Activates the most recently fetched Product configs
   static Future<void> activate() async {
     return await _dartToNativeMethodChannel.invokeMethod('activate', {});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Fetches and then activates the fetched Product configs.
   static Future<void> fetchAndActivate() async {
     return await _dartToNativeMethodChannel.invokeMethod('fetchAndActivate', {});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Sets the minimum interval between successive fetch calls.
   static Future<void> setMinimumFetchIntervalInSeconds(int interval) async {
     return await _dartToNativeMethodChannel.invokeMethod(
         'setMinimumFetchIntervalInSeconds', {'interval': interval});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Returns the last fetched timestamp in millis.
   static Future<int?> getLastFetchTimeStampInMillis() async {
     return await _dartToNativeMethodChannel.invokeMethod('getLastFetchTimeStampInMillis', {});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Returns the parameter value for the given key as a String.
   static Future<String?> getProductConfigString(String key) async {
     return await _dartToNativeMethodChannel.invokeMethod('getString', {'key': key});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
+
   ///Returns the parameter value for the given key as a boolean.
   static Future<bool?> getProductConfigBoolean(String key) async {
     return await _dartToNativeMethodChannel.invokeMethod('getBoolean', {'key': key});
   }
 
+  @Deprecated(
+      "This method is deprecated since version 1.6.0 and will be removed in the future versions of this SDK.")
```

Comment:
@akashvercetti sorry for the confusion, this should be 1.7.0
I thought latest version is 1.5.6 but it is actually 1.6.1